htdocs/compta/facture.php

```diff
@@ -3990,7 +3990,7 @@
 			{
 				if (! $objectidnext)
 				{
-					print '<div class="inline-block divButAction"><a class="butAction" href="' . $_SERVER['PHP_SELF'] . '?socid=' . $object->socid .'&amp;fac_avoir=' . $object->id . '&amp;action=create&amp;type=2">' . $langs->trans("CreateCreditNote") . '</a></div>';
+					print '<div class="inline-block divButAction"><a class="butAction" href="' . $_SERVER['PHP_SELF'] . '?socid=' . $object->socid .'&amp;fac_avoir=' . $object->id . '&amp;action=create&amp;type=2&amp;projectid='.$object->fk_project.'">' . $langs->trans("CreateCreditNote") . '</a></div>';
```

Comment:
What about if object->projectid is not defined ?
I hat to have linke with &param= and no value because it create troubles with pagination.